### PR TITLE
GCW-1929 download banner for web

### PR DESCRIPTION
**Pull request by abulsayyad123**
Hi Team,
This is PR for JIRA ticket GCW-1929- Add "download/open App" banner to web versions of each app.
I have added this change in this repo which will reflect the change for both DONOR and ADMIN app.

Please review the PR and let me know if any change is required.
Thanks.

### CODE REVIEW

File: app/controllers/application.js [hunk 1]
```diff
@@ -8,13 +8,21 @@ export default Ember.Controller.extend({
...
+  ios_app_id: config.APP.IOS_APP_ID,
 
   initSubscriptions: Ember.on('init', function() {
     if (this.session.get("isLoggedIn")) {
       this.send('setSubscriptions');
     }
   }),
 
+  appIcon: Ember.computed('config', function() {
+    if (config.APP.NAME === 'app.goodcity') {
```
Comment (steveyken) [inline app/controllers/application.js]: Would it be better to have config.APP.TITLE defined?

Comment (abulsayyad123) [reply app/controllers/application.js]: @steveyken , you mean taking `config.APP.NAME` in a separate variable like `let appTitle = config.APP.NAME` and using it in if condition or declaring different field `config.APP.TITLE` in environment file ?

Comment (steveyken) [reply app/controllers/application.js]: The latter one - define `config.APP.TITLE` in environment file

Comment (abulsayyad123) [reply app/controllers/application.js]: Added 👍

**Pull request merged**